care/facility/migrations/0385_patientconsultation_is_readmission.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 4.2.2 on 2023-09-01 16:22
+
+from datetime import timedelta
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("facility", "0384_patientconsultation_verified_by"),
+    ]
+
+    def fill_is_readmission(apps, schema_editor):
+        PatientConsultation = apps.get_model("facility", "PatientConsultation")
+
+        bulk_update_list = []
+        qs = PatientConsultation.objects.filter(
+            suggestion="A", admission_date__isnull=False
```

Comment:
```suggestion
            suggestion="A"
```
we should include ongoing consultations as well

Reply:
ongoing consultations too will have `admission_date`. What ongoing admissions won't have is a `discharge_date`. So isn't the case already handled?

Also, check the screenshot in the front-end PR. The latest ongoing consultation already has readmission=True set.

<img width="1822" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/7f6c0072-f442-4c4d-8048-1ce48ea30521">